package.json

```diff
@@ -18,52 +18,35 @@
   },
   "license": "BSD-3-Clause",
   "main": "lib/index.js",
+  "module": "lib/index.js",
```

Comment:
Until this package starts doing ESM thing, I don't think this is needed.

Reply:
Thanks, removed.